TPP.Core/Chat/TwitchChatQueue.cs

```diff
@@ -97,47 +95,57 @@ private async Task ProcessOne()
 
         private async Task Send(OutgoingMessage message)
         {
+            TwitchAPI twitchApi = await _twitchApiProvider.Get();
             if (message is OutgoingMessage.Chat chat)
-                await _twitchClient.SendMessageAsync(chat.Channel, chat.Message);
+                await twitchApi.Helix.Chat.SendChatMessage(chat.ChannelId, _senderUserId, chat.Message);
             else if (message is OutgoingMessage.Reply reply)
-                await _twitchClient.SendReplyAsync(reply.Channel, replyToId: reply.ReplyToId, message: reply.Message);
+                // TODO https://github.com/TwitchLib/TwitchLib.Api/pull/386
+                await twitchApi.Helix.Chat.SendChatMessage(reply.ChannelId, _senderUserId, reply.Message /*, replyParentMessageId: reply.ReplyToId*/);
```

Comment:
Need to fix this bit if/when https://github.com/TwitchLib/TwitchLib.Api/pull/386 is merged and available on nuget via a prerelease

Reply:
I've disabled twitch-native replies for now in 620b440f1c0016218159e7093fbf315bcc4b820c until we can use the API for them again. So we need to also revert 620b440f1c0016218159e7093fbf315bcc4b820c once this is fixed